api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsWithHighViralLoad.java

```diff
@@ -0,0 +1,115 @@
+package org.openmrs.module.eptsreports.reporting.reports;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Properties;
+import org.openmrs.Location;
+import org.openmrs.module.eptsreports.reporting.library.datasets.DatimCodeDatasetDefinition;
+import org.openmrs.module.eptsreports.reporting.library.datasets.ListOfPatientsWithHighViralLoadCohortDataset;
+import org.openmrs.module.eptsreports.reporting.library.datasets.SemanaCorrenteDatasetOfHighViralLoadCohortDataset;
+import org.openmrs.module.eptsreports.reporting.library.datasets.SismaCodeDatasetDefinition;
+import org.openmrs.module.eptsreports.reporting.reports.manager.EptsDataExportManager;
+import org.openmrs.module.reporting.ReportingException;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Deprecated
+public class SetupListOfPatientsWithHighViralLoad extends EptsDataExportManager {
+
+  private SemanaCorrenteDatasetOfHighViralLoadCohortDataset
+      semanaCorrenteDatasetOfHighViralLoadCohortDataset;
+  private ListOfPatientsWithHighViralLoadCohortDataset listOfPatientsWithHighViralLoadCohortDataset;
+
+  @Autowired
+  public SetupListOfPatientsWithHighViralLoad(
+      SemanaCorrenteDatasetOfHighViralLoadCohortDataset
+          semanaCorrenteDatasetOfHighViralLoadCohortDataset,
+      ListOfPatientsWithHighViralLoadCohortDataset listOfPatientsWithHighViralLoadCohortDataset) {
+    this.semanaCorrenteDatasetOfHighViralLoadCohortDataset =
+        semanaCorrenteDatasetOfHighViralLoadCohortDataset;
+    this.listOfPatientsWithHighViralLoadCohortDataset =
+        listOfPatientsWithHighViralLoadCohortDataset;
+  }
+
+  @Override
+  public String getUuid() {
+    return "471b83a4-7ae3-11ed-b983-67166207dca4";
+  }
+
+  @Override
+  public String getName() {
+    return "List of Patients with High Viral Load Cohort Report";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Lista de Pacientes com Carga Viral Alta";
```

Comment:
Update the description of the report for **Lista de Seguimento De Pacientes Com Carga Viral Não Suprimida** FR1